initfiles/componentfiles/configxml/mysqlserver.xsd

```diff
@@ -116,12 +116,19 @@
                     </xs:appinfo>
                 </xs:annotation>
             </xs:attribute>
-            <xs:attribute name="computer" type="computerType" use="required">
+            <xs:attribute name="computer" type="computerType" use="optional">
                 <xs:annotation>
                     <xs:appinfo>
                         <tooltip>The computer on which the MySQL database is deployed.</tooltip>
                         <autogenforwizard>1</autogenforwizard>
                     </xs:appinfo>
+                </xs:annotation> 
+            </xs:attribute>
+            <xs:attribute name="host" type="xs:string" use="optional">
```

Comment:
The use is in the LN repository where the logging service is configured with mysql server. Will be in different ticket and pull request.